tests/unit/backend/data/test_GroceryService.py

```diff
@@ -160,7 +162,7 @@ def check_workspace_name_has_extra_token(self, wsname_1: str, wsname_2: str, ext
 
     ## TESTS OF FILENAME METHODS
 
-    @mock.patch.object(LocalDataService, "getIPTS")
+    @mock.patch.object(DataService, "getIPTS")
```

Comment:
Is this really what you want to patch?  Do you need to patch it?  It doesn't actually have a "getIPTS" object.

Reply:
I think its easier having a mock in place rather then risking calls bleed into other classes by default

Reply:
Are these changes largely just redirecting the patches to path a mock?  Because I had another PR to remove the patches entirely

Reply:
Yes, but I had made these prior to that pr, Id rather not have to redux my changes